docs/troubleshooting.md

```diff
@@ -37,3 +38,10 @@ you need to properly configure CORS on your server: https://github.com/videojs/h
 Some browsers support native playback of certain streaming formats. By default, we defer to the native players. However, this means that features specific to videojs-http-streaming will not be available.
 
 Solution: use videojs-http-streaming based playback on those devices: https://github.com/videojs/http-streaming#overridenative
+
+## media_err_decode error on Desktop Safari
+
+This error may occur if the source manifest has `CLOSED-CAPTIONS=NONE` and an external text track is loaded into the player. Safari does not allow the inclusion any captions if the manifest indicates that captions will not be provided.
```

Comment:
We may also want to mention that MEDIA_ERR_DECODE can happen for a number of reasons, as it is particularly common for misconfigured content, the captions being just one of them.